data/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
```suggestion

```

Reply:
It is the directory where the data are downloaded - I do not want to see any of it.

@Borda  Why do you want to remove `*`?

Reply:
Directory with data is fine, but here is just a * meaning anything, right?

Reply:
Right - `*` means anything - the semantics here -  under the `data` directory you can paste/download anything and it will be ignored.
Note that the `.gitignore` is in the data subdirectory.

I can change it to the default data_root under the data directory for covid19 if it bothers you 
i.e. `covid-chestxray-dataset`

Reply:
ohh sorry, I missed that it not in the root
so then rather place this folder in the root ignore

Reply:
@Borda ok I will add .gitkeep and move the ignore statement to the global gitignore

Reply:
done